scripts/basic-workflow-tests.sh

```diff
@@ -133,13 +133,24 @@ test_generate() {
 test_train() {
     task Train the model
 
+    # Reduce testfile to 1 sample, generating from the new model is extreemly slow in CI
+    # generating only a single test should be enough as we're not verifying the output
+    TESTFILE=$(set +f ; ls -tr generated/test_*jsonl | tail -n 1)
```

Comment:
(drive by comment) 

would this be better as 
```
TESTFILE=$(set +f ; ls -1tr generated/test_*jsonl | tail -n 1)
```
or even simpler

```
TESTFILE=$(set +f ; ls -1t generated/test_*jsonl | head -n 1)
```
The script says it can be run anywhere ilab is run, so it feels like `-1` it might be more resilient to `ilab generate` getting run with multiple models ?